model_servers/common/Makefile.common

```diff
@@ -10,7 +10,7 @@ endif
 
 .PHONY: build
 build:
-	podman build --squash-all --build-arg $(PORT) -t $(IMAGE) . -f base/Containerfile
+	podman build --squash-all --build-arg PORT=$(PORT) -t $(IMAGE) . -f base/Containerfile
```

Comment:
This will blow up with PORT is not defined or will at least pass in a PORT="" line.

Reply:
It is defined in every local makefile which calls the common one, so no matter what it will have an argument here